corehq/util/doc_processor/couch.py

```diff
@@ -25,10 +25,12 @@ def resumable_docs_by_type_iterator(db, doc_types, iteration_key, chunk_size=100
     """
 
     def data_function(**view_kwargs):
+        view_kwargs["limit"] = chunk_size
         return db.view('all_docs/by_doc_type', **view_kwargs)
 
     keys = [[doc_type] for doc_type in doc_types]
-    args_provider = MultiKeyViewArgsProvider(keys, include_docs=True, chunk_size=chunk_size)
+    args_provider = MultiKeyViewArgsProvider(keys, include_docs=True)
+    args_provider.initial_view_kwargs.pop("limit")
```

Comment:
Not a major reason, just cleaning up irrelevant junk. If it's not popped the default value (1000) gets saved with the state in the meta db. I figured it would be confusing to anyone looking at the state to see that value in there even though it's not the real chunk size.